src/components/Navigation.tsx

```diff
@@ -1,43 +1,94 @@
 import Image from 'next/image';
+import { useEffect, useState } from 'react';
 
-import { MyLink } from './MyLink';
+function useWindowSize() {
+  const [windowSize, setWindowSize] = useState({
+    width: 0,
+    height: 0,
+  });
+  useEffect(() => {
+    function handleResize() {
+      setWindowSize({
+        width: window.innerWidth,
+        height: window.innerHeight,
+      });
+    }
+    window.addEventListener('resize', handleResize);
+    handleResize();
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
+  return windowSize;
+}
+
+export function Navigation({
+  handleShowMobileTree,
+}: {
+  handleShowMobileTree: (width: number) => void;
+}) {
+  const { width } = useWindowSize();
+
+  useEffect(() => {
+    handleShowMobileTree(width);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [width]);
 
-export function Navigation() {
   return (
-    <nav
-      className="mx-auto flex h-20 items-center
-      justify-between border-b border-gray-600 pt-6 pb-4"
-    >
-      <section className="flex items-center">
+    <nav className="mx-auto flex h-20 items-center justify-between px-8 pt-6 pb-4 sm:px-0">
+      <section className="flex cursor-pointer items-center gap-4">
         <Image
-          className="cursor-pointer"
-          src="/static/svg/deth-tools.svg"
-          width={40}
-          height={40}
+          className=""
+          src="/static/svg/logo.svg"
+          width={36}
+          height={36}
           alt="deth tools logo"
         />
-        <h1 className="cursor-pointer font-syne text-4xl text-gray-900">
-          Deth Tools
-        </h1>
+        {width > 480 && (
+          <Image
+            className=""
+            src="/static/svg/deth-tools.svg"
+            width={164}
+            height={164}
+            alt="deth tools logo"
+          />
+        )}
+        <p> WIP </p>
       </section>
 
       <section className="flex items-center gap-4">
-        <MyLink href={'/'}>
-          <button
-            className="border border-gray-900 py-2 px-4
-          hover:bg-black hover:text-white"
-          >
-            Let's get started
-          </button>
-        </MyLink>
-
-        <Image
-          className="cursor-pointer rounded-full hover:bg-blue-200"
-          src="/static/svg/github.svg"
-          width={26}
-          height={26}
-          alt="deth tools github logo"
-        />
+        {width > 480 ? (
+          <section className="flex gap-3">
+            <Image
```

Comment:
Do we really want to use Next Image for SVGs? Does it do anything? I'm not sure how SGVs tie into the image optimization pipeline.